### PR TITLE
[TASK] Tray icon (DC-49), [TASK] Multiple tx file DnD support (DC-76)

### PR DESCRIPTION
**Pull request by Mesrop**

### CODE REVIEW

**Comment by vhpoet** [comment]
@Mesrop Please put nw.js related stuff in the entry.js file, or maybe even create a new file for nw.js stuff and include it in entry.js

**Comment by Mesrop** [comment]
@vhpoet I think now we have the most optimal version.

**Comment by vhpoet** [comment]
@Mesrop as discussed, please wrap nw.js stuff into services.

**Comment by Mesrop** [comment]
@vhpoet pls look at this again.

File: src/js/tabs/balance.js [hunk 1]
```diff
@@ -15,20 +14,22 @@ BalanceTab.prototype.mainMenu = 'wallet';
...
-  {
-    if (!$id.loginStatus) $id.goId();
-  }]);
+
+  module.controller('BalanceCtrl', ['$rootScope', 'rpId', 'rpNW',
+    function($scope, $id, rpNW) {
+      if (!$id.loginStatus) {
+        $id.goId();
+      } else {
+        rpNW.initTray();
```
Comment (vhpoet) [inline src/js/tabs/balance.js]: You already called this in entry.js

Comment (vhpoet) [inline src/js/tabs/balance.js]: @Mesrop what's the status here?

Comment (Mesrop) [inline src/js/tabs/balance.js]: @vhpoet calling initTray again reinitializes the existing tray with new menu depending on the user's authorization status.

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**